examples/seismic/model.py

```diff
@@ -57,21 +57,25 @@ def __init__(self, origin, spacing, vp, nbpml=0, dtype=np.float32,
         self.spacing = spacing
         self.nbpml = nbpml
         self.dtype = dtype
-
+        # Generate a random id with the model.
+        # This is needed for the Gradient test that uses two different models
+        # Model(m) and Model(m0) that uses the same value of m without id
```

Comment:
There is still some problems with DSE, but can be part of another PR I guess
The failing TRAVIS is a bug when `compiler=None`, I'll fix it